source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -5906,13 +5906,17 @@ export abstract class StylableCardElementContainer extends CardElementContainer
     }
 
     protected getHasBackground(): boolean {
+        return this.getHasPadding(true);
```

Comment:
should this always be true or should it depend upon GlobalSettings.addPaddingToContainerWithBackgroundImage value?